data/mods/gen2stadium2/scripts.ts

```diff
@@ -329,25 +328,30 @@ export const Scripts: ModdedBattleScriptsData = {
 				level = move.allies[0].level;
 			}
 
-			let attacker = source;
-			const defender = target;
-			if (move.useTargetOffensive) attacker = target;
-			let atkType: StatIDExceptHP = (move.category === 'Physical') ? 'atk' : 'spa';
-			const defType: StatIDExceptHP = (move.defensiveCategory === 'Physical') ? 'def' : 'spd';
-			if (move.useSourceDefensiveAsOffensive) atkType = defType;
+			const attacker = move.useBaseOffensiveStatAndBoosts?.includes('target') ? target : source;
+			const defender = move.useBaseDefensiveStatAndBoosts?.includes('source') ? source : target;
+
+			let atkType: StatIDExceptHP = move.category === 'Physical' ? 'atk' : 'spa';
+			if (move.useBaseOffensiveStatAndBoosts) {
+				atkType = move.useBaseOffensiveStatAndBoosts[1];
+			}
+			let defType: StatIDExceptHP = move.category === 'Physical' ? 'def' : 'spd';
+			if (move.useBaseDefensiveStatAndBoosts) {
+				defType = move.useBaseDefensiveStatAndBoosts[1];
+			}
 			let unboosted = false;
 			let noburndrop = false;
 
 			if (isCrit) {
 				if (!suppressMessages) this.battle.add('-crit', target);
 				// Stat level modifications are ignored if they are neutral to or favour the defender.
 				// Reflect and Light Screen defensive boosts are only ignored if stat level modifications were also ignored as a result of that.
-				if (attacker.boosts[atkType] <= defender.boosts[defType]) {
+				if ((attacker.boosts[atkType] <= defender.boosts[defType])) {
```

Comment:
?

Reply:
Guessing this is just a typo.

```suggestion
				if (attacker.boosts[atkType] <= defender.boosts[defType]) {
```

Reply:
Guessing this is just a typo.

```suggestion
				if (attacker.boosts[atkType] <= defender.boosts[defType]) {
```

Reply:
Left overs from removing the hp stats from the override stat support. Good catch